GiniVision/Classes/CameraViewController.swift

```diff
@@ -245,18 +257,27 @@ public typealias CameraScreenFailureBlock = (_ error: GiniVisionError) -> ()
         camera?.stop()
     }
     
+    public override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+        
+        if shouldShowFileImportToolTip {
+            showFileImportTip()
+        }
+    }
+    
     public override func viewWillTransition(to size: CGSize, with coordinator: UIViewControllerTransitionCoordinator) {
         super.viewWillTransition(to: size, with: coordinator)
-        
+
         coordinator.animate(alongsideTransition: { [weak self] _ in
             guard let `self` = self else {
                 return 
             }
             (self.previewView.layer as? AVCaptureVideoPreviewLayer)?.connection?.videoOrientation = self.getVideoOrientation()
-            
+            self.toolTipView?.arrangeViews()
```

Comment:
`arrangeViews ` as a name doesn't really give me a lot of information about what happens there. I assume it re-layouts the view with the new size?

Reply:
It position/arrange views in the correct position.